app/models/journey.rb

```diff
@@ -36,7 +36,6 @@ class Journey < ApplicationRecord
   belongs_to :from_location, class_name: 'Location'
   belongs_to :to_location, class_name: 'Location'
 
-  has_many :events, as: :eventable, dependent: :destroy # NB: polymorphic association
   has_many :generic_events, as: :eventable, dependent: :destroy # NB: polymorphic association
```

Comment:
I think the comment here can be dropped now - it's not adding any value TBH

Reply:
Not sure it ever added value, tbh :)